fxgl/src/main/kotlin/com/almasb/fxgl/dsl/FXGL.kt

```diff
@@ -589,6 +606,39 @@ class FXGL private constructor() { companion object {
                 .also { addUINode(it, x, y) }
     }
 
+    /**
+     * Returns new Text initialized with property value, whose name was passed to the function
+     * @param propertyName name of the property in PropertyMap
+     * @param x x coordinate
+     * @param y y coordinate
+     * @throws IllegalArgumentException if property was not found in PropertyMap
+     * @return Text()
+     */
+    @JvmStatic fun addPropText(propertyName: String, x: Double, y: Double) : Text {
+        if (!getWorldProperties().exists(propertyName))
+            throw IllegalArgumentException("Property with name '$propertyName' does not exists in PropertyMap.")
+
+        return getUIFactoryService().newText(getPropertyValueAsString(propertyName))
+                    .also { addUINode(it, x, y) }
+    }
+
+    /**
+     * Returns property value as String
+     * @param propertyName name of the property in PropertyMap
+     * @return string value of property
+     * @throws IllegalArgumentException if property has unknown type (unknown in terms of 'when' clauses)
+     */
+    @JvmStatic private fun getPropertyValueAsString(propertyName: String): String? {
```

Comment:
This should return `StringBinding` or `StringProperty`, i.e. the new text object's `textProperty()` needs to be bound to a property variable, so that when the variable value changes, the text is updated too.

Reply:
I think the fix is to just remove the `.get()` calls?

Reply:
changed return type to StringExpression for  `newText(StringExpression textBinding)`